easy_my_coop_taxshelter_report/models/tax_shelter_declaration.py

```diff
@@ -281,7 +283,7 @@ def _compute_amounts(self):
             certificate.total_amount_resold = total_amount_resold
             certificate.total_amount = certificate.total_amount_previously_subscribed + certificate.total_amount_subscribed + certificate.total_amount_resold + certificate.total_amount_transfered
 
-    @api.multi
+    @api.depends('lines')

```

Comment:
Those fields are not stored. they're only computed. So I think depends is useless here.

Reply:
Not sure if it can be removed. Should be tested. Had no time to properly test it.